requirements.txt

```diff
@@ -0,0 +1 @@
+awscli==1.7.25
```

Comment:
It's a great idea but it creates an external dependency on the heroku build pack type.

On Mon, May 4, 2015 at 11:22 PM, Ken Liu <notifications@github.com> wrote:

>> @@ -0,0 +1 @@
>> +awscli==1.7.25
> awscli is a python application so it is fitting to use `pip` to manage the dependency. it's a good idea to let Heroku handle fetching the dependency. less things to break.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/kbaum/heroku-database-backups/pull/5/files#r29641935